runner/conmon_test/suite_test.go

```diff
@@ -60,7 +60,7 @@ func getConmonOutputGivenOptions(options ...conmon.ConmonOption) (string, string
 
 	alljournalout, err := getConmonJournalOutput(pid, -1)
 	Expect(err).To(BeNil())
-	fmt.Fprintf(GinkgoWriter, alljournalout+"\n")
+	fmt.Fprintf(GinkgoWriter, "%s\n", alljournalout)
```

Comment:
Perhaps it's better/simpler to use `fmt.Fprintln` here?

```suggestion
	fmt.Fprintln(GinkgoWriter, alljournalout)
```